website/docs/commands/basics.md

```diff
@@ -181,6 +181,20 @@ echo 'println("Hello")' | scala-cli -
 # Hello
 ```
 
+## ScalaCLI version
+
+`scala-cli` can also run the other version of `scala-cli`, it may be helpful to test unreleased scala-cli functionality.
+
+```bash ignore
```

Comment:
Can we use `ChainedSnippets` and use expected outputs as in https://github.com/VirtusLab/scala-cli/blob/master/website/docs/getting_started.md ?

Reply:
Also why we do need to use `ignore`?